config/helm/aws-node-termination-handler/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: aws-node-termination-handler
 description: A Helm chart for the AWS Node Termination Handler
-version: 0.14.2
+version: 0.15.1
```

Comment:
how did you arrive at this version number?

Reply:
I manually ran `make ekscharts-sync` and saw that the automatically created PR to `aws/charts` failed because the chart version hadn't been bumped, so I went ahead and bumped it. Not sure if this happens automatically in some build process step.

Since this is not a bug fix PR I bumped the minor version, however now I realise I should've used `0.15.0`. In any case IDK how this decision is made so happy to set the value you see fit.

Reply:
Yeahhh it's not very clear how we bump this--both in documentation and in the commit history, but appreciate that you tried 😅

It'll be easier if our team owns incrementing these chart versions, so **no need for you to make changes to this file in this pr**

#### How we bump versions
If you are interested in the details, we run `make release-prep-<patch,minor,major>` target and the script it calls will bump app and chart versions based on the release type. The target you ran `make ekscharts-sync` is used during the release process to auto-cut a PR to [eks-charts](https://github.com/aws/eks-charts) repo to keep the versions in sync. [Example PR](https://github.com/aws/eks-charts/pull/484)